### PR TITLE
Fixes #14746 - Template kinds translated in provisioning template for…

### PR DESCRIPTION
**Pull request by xprazak2**
…m dropdown

### CODE REVIEW

File: app/models/template_kind.rb [hunk 1]
```diff
@@ -27,4 +27,8 @@ def self.add_to_jar(hash)
   def to_s
     TemplateKind.jar[name] || name
   end
+
```
Comment (dLobatog) [inline app/models/template_kind.rb]: This is kind of unnecessary, as it forces changing the view too. Can you change `to_s` to do this?

``` ruby
_(TemplateKind.jar[name] || name)
```

**Comment by dLobatog** [comment]
@xprazak2 Thanks for addressing this, take a look at the comment inline, it'd be easy to fix!

**Comment by dLobatog** [comment]
@xprazak2 This needs a rebase against latest develop. Also please take a look at the commit message, we've enforced more strict rules.

**Comment by xprazak2** [comment]
[test]

**Comment by dLobatog** [comment]
Merged as 67a2fe79000f14d7963ec92b34c10d4f7136eff5, thanks @xprazak2!

**Pull request closed without merging**